### DISCUSSION TITLE
Custom select when using batch loading.

### DISCUSSION THREAD

**Discussion by goni098** [SeaQL/sea-orm#1768] [Q&A]
For example :
```
let cakes: Vec<cake::Model> = Cake::find()
    .select_only()
    .column(cake::Column::Name)
    .column(cake::Column::Id)
    .all(db)
    .into_model::<CustomStruct>()
   .await?;

let fruits: Vec<Vec<fruit::Model>> = cakes.load_many(Fruit, db).await?;
```
I can not use load_many method on cakes. Is there any to resolve it and select  partial both fruits and cakes table as well ?

**Comment by billy1624** [comment]
Maybe this is helpful? https://www.sea-ql.org/sea-orm-cookbook/001-how-to-load-nested-relation.html
